telosnetwork/telos-wallet
# adding new notification options

# Fixes #253

## Description
This PR has two temporal entries in the side menu to manually trigger a testing success and error message with the new styling. This implementation was needed for showing feedback on the new sending page task.

## Test scenarios
- got to [this link](https://deploy-preview-255--wallet-staging.netlify.app/)
- authenticate in EVM
- open the left side menu (if closed)
- press the Success Notification button
  - A successful transaction message should appear for 5 sec and go
  - the styling should fit the design
  - if you press the see transaction button it should open a new browser tab with the corresponding browser on the tr page
- press the failure Notification button
  - A successful transaction message should appear for 5 sec and go
  - the styling should fit the design

## Note
once this PR gets approved, the side menu buttons should be removed before merging

## Screenshots
![image](https://user-images.githubusercontent.com/4420760/232168019-e8b5473b-c0fa-44df-87df-5c963a082d5d.png)
![image](https://user-images.githubusercontent.com/4420760/232168052-ee432b6b-9eef-4d53-9b54-321e837bbdc7.png)


## Checklist:
